admin_views/management/commands/admin_views_install_templates.py

```diff
@@ -14,7 +16,13 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         current_dir = os.path.dirname(__file__)
-        template_dirs = settings.TEMPLATE_DIRS
+        if StrictVersion(django.get_version()) < StrictVersion('1.8'):
+            template_dirs = settings.TEMPLATE_DIRS
+        else:
+            template_dirs = []
+            for template_engine in settings.TEMPLATES:
+                if template_engine['BACKEND'] == 'django.template.backends.django.DjangoTemplates':
+                    template_dirs = template_dirs + template_engine['DIRS']
```

Comment:
Why not `break` if a template directory for Django's template engine has been found?